### PR TITLE
Fixed a couple of typos, word 'deliberately' and database_resolver_context class name. [ci skip]

### PR DESCRIPTION
**Pull request by akshaymohite**
### Summary

- `deliberatly` -> `deliberately`
- `MyCookieResovler` -> `MyCookieResolver`

### CODE REVIEW

File: guides/source/active_record_multiple_databases.md [hunk 1]
```diff
@@ -247,13 +247,13 @@ The `database` argument for `connected_to` will take a symbol or a config hash.
 
 Note that `connected_to` with a role will look up an existing connection and switch
 using the connection specification name. This means that if you pass an unknown role
-like `connected_to(role: :nonexistent)` you will get an error like that says
+like `connected_to(role: :nonexistent)` you will get an error that says
```
Comment (kamipo) [inline guides/source/active_record_multiple_databases.md]: Should be removed this "like"?

Comment (akshaymohite) [reply guides/source/active_record_multiple_databases.md]: @kamipo 

`like` seems inappropriate at the place it was used. It needs to be associated with the error message that follows.

How about rewording as,
```diff
- you will get an error like that says `ActiveRecord::ConnectionNotEstablished (No connection pool with 'AnimalsBase' found for the 'nonexistent' role.)`
+ you will get an error that says something like `ActiveRecord::ConnectionNotEstablished (No connection pool with 'AnimalsBase' found for the 'nonexistent' role.)` 
```
instead of removing `like` word?

Comment (vipulnsward) [reply guides/source/active_record_multiple_databases.md]: lgtm
